xla/service/gpu/gpu_compiler.cc

```diff
@@ -655,20 +656,22 @@ absl::Status RunSPMDPasses(
   const int64_t num_partitions = hlo_module->config().num_partitions();
   if (num_partitions > 1 && hlo_module->config().use_spmd_partitioning()) {
     HloPassPipeline spmd_pipeline("spmd-partitioner");
-    AddSPMDPasses(hlo_module, layout_insensitive_algsimp_opts,
-                  gpu_target_config.device_description.gpu_compute_capability(),
-                  spmd_pipeline,
+    AddSPMDPasses(
```

Comment:
There are only formatting changes here. Can you avoid formatting changes in the next PRs to ease review? All of this will be formatted on import.